src/garage/torch/policies/continuous_mlp_policy.py

```diff
@@ -0,0 +1,70 @@
+"""
+This modules creates a continuous MLP policy network.
+
+A continuous MLP network can be used as policy method in different RL
+algorithms. It accepts an observation of the environment and predicts an
+action.
+"""
+import akro
+import torch
+
+from garage.misc.overrides import overrides
+from garage.tf.policies.base import Policy
+
+
+class ContinuousMLPPolicy(Policy):
+    """
+    This class implements a policy network.
+
+    The policy network selects action based on the state of the environment.
+    It uses neural nets to fit the function of pi(s).
+    """
+
+    def __init__(self,
+                 env_spec,
+                 nn_module,
+                 name='ContinuousMLPPolicy',
+                 input_include_goal=False):
+        """
+        Initialize class with multiple attributes.
+
+        Args:
+            env_spec():
+            nn_module():
+                A PyTorch module.
+            name(str, optional):
+                A str contains the name of the policy.
+        """
+        assert isinstance(env_spec.action_space, akro.Box)
+
+        super().__init__(env_spec)
+
+        self._env_spec = env_spec
+        self._nn_module = nn_module
+        self._name = name
+        if input_include_goal:
```

Comment:
this is some technical debt from the HER implementation in TF -- please remove and do it the right way in Torch (the goal should be added to the input before it gets to this module)

Reply:
basically, you just have to remove this case and not consider it.